GliaWidgets/Component/Button/Action/ActionButtonStyle.swift

```diff
@@ -62,10 +62,10 @@ public struct ActionButtonStyle: Equatable {
         titleColor: UIColor,
         textStyle: UIFont.TextStyle = .body,
         backgroundColor: ColorType,
-        cornerRaidus: CGFloat? = 4.0,
-        shadowOffset: CGSize? = CGSize(width: 0.0, height: 2.0),
-        shadowColor: UIColor? = UIColor.black,
-        shadowRadius: CGFloat? = 2.0,
+        cornerRaidus: CGFloat = 4.0,
```

Comment:
Same in all of these. If they don't send anything then the default value would be used, but in the previous version, they could send `nil`, which they can't now, so it's a breaking change as well.

Reply:
fixed